daemon/execdriver/windows/exec.go

```diff
@@ -114,13 +50,23 @@ func (d *Driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessCo
 	logrus.Debugln("commandLine: ", createProcessParms.CommandLine)
 
 	// Start the command running in the container.
-	pid, err = hcsshim.CreateProcessInComputeSystem(c.ID, createProcessParms)
-
+	pid, stdin, stdout, stderr, err := hcsshim.CreateProcessInComputeSystem(c.ID, pipes.Stdin != nil, true, !processConfig.Tty, createProcessParms)
 	if err != nil {
 		logrus.Errorf("CreateProcessInComputeSystem() failed %s", err)
 		return -1, err
 	}
 
+	// Now that the process has been launched, begin copying data to and from the named pipes for the std handles.
+	if stdin != nil {
+		stdinCopy(stdin, pipes.Stdin)
+	}
+	if stdout != nil {
+		stdouterrCopy(stdout, "stdout", pipes.Stdout)
```

Comment:
@jhowardmsft I don't like that these functions are asynchronous. Should have `start` or `setup` in the name. But more importantly, why do we need to copy stdout and stderr? (we don't seem to do it on the linux codepath)